aten/src/TH/THGeneral.h.in

```diff
@@ -53,11 +61,12 @@ typedef struct {
 
 TH_API double THLog1p(const double x);
 TH_API THDescBuff _THSizeDesc(const int64_t *size, const int64_t ndim);
-TH_API void _THError(const char *file, const int line, const char *fmt, ...);
+TH_API TH_NO_RETURN void _THError(const char *file, const int line, const char *fmt, ...);
 TH_API void _THAssertionFailed(const char *file, const int line, const char *exp, const char *fmt, ...);
 TH_API void THSetErrorHandler(THErrorHandlerFunction new_handler, void *data);
 TH_API void THSetDefaultErrorHandler(THErrorHandlerFunction new_handler, void *data);
 TH_API void _THArgCheck(const char *file, int line, int condition, int argNumber, const char *fmt, ...);
+TH_API void _THArgError(const char *file, int line, int argNumber, const char *fmt, ...);
```

Comment:
Is this actually implemented?

Reply:
It's a leftover of some temporary things, it is not implemented but not used either. Will fix.